src/components/LoanCards/LoanTags/LoanCallouts.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+	<div class="tw-line-clamp-1">
```

Comment:
It looks like clamping is just removing the second pill if there's wrap. My guess is that we instead want to truncate the pills themselves or something along those lines.
![image](https://user-images.githubusercontent.com/16867161/217402798-030b936b-47bc-400f-8ff9-80e699765bf3.png)